packager/live_packager_test.cc

```diff
@@ -1499,4 +1534,33 @@ INSTANTIATE_TEST_CASE_P(
             Status(error::INVALID_ARGUMENT, "Stream not available"),
             0,
         }));
+
+TEST_F(LivePackagerBaseTest, TestCmafTimedText) {
+  std::vector<uint8_t> segment_buffer =
+      ReadTestDataFile("timed_text/cmaf/text_cmaf_fragment_343138171.vtt");
+  ASSERT_FALSE(segment_buffer.empty());
+
+  SegmentData media_seg(segment_buffer.data(), segment_buffer.size());
+
+  FullSegmentBuffer out;
+
+  LiveConfig live_config;
+  live_config.format = LiveConfig::OutputFormat::VTTMP4;
+  live_config.track_type = LiveConfig::TrackType::TEXT;
+  live_config.protection_scheme = LiveConfig::EncryptionScheme::NONE;
+  live_config.segment_number = 343138171;
+  live_config.timed_text_decode_time = (154412176500000 / 90000) * 1000;
```

Comment:
Could you please leave a comment here to indicate the intent of this test, i.e., to ensure a large decode time (> 32 bits) can be properly handled?